public void testChildhoodInfectionException() throws Exception {
    try {
        evilDAO.hadChildhoodInfection(0L, 0.0);
        fail("DBException should have been thrown");
    } catch (DBException e) {
        assertEquals(EvilDAOFactory.MESSAGE, e.getSQLException().getMessage());
    }
}